src/sdk.ts

```diff
@@ -12,10 +12,22 @@ import { CapacitorScope } from './scope';
 import { NativeTransport } from './transports/native';
 import { NATIVE } from './wrapper';
 
-const DEFAULT_OPTIONS: CapacitorOptions = {
+export const DEFAULT_OPTIONS_WEB: CapacitorOptions = {
+  enableNativeNagger: false,
+  enableNative: false,
+
+  // Use the browser's session tracking instrumentation on web.
+  enableAutoSessionTracking: false,
+  autoSessionTracking: true,
```

Comment:
Can we rename these somehow? It's weird that they feel like the same exact option right now.

Reply:
How about we have it be `enableNativeSessionTracking` and `enableBrowserSessionTracking`? The init call here in cap would just convert the two into the respective options for the browser sdk and the native bridge? cc @bruno-garcia

Reply:
I like splitting it up that way. 

Should we even refer to native at all, maybe just call it `enableMobileSessionTracking`? I think we should do whatever we think will make it easier for users to understand.

Reply:
@AbhiPrasad In that sense, let's just have a single `enableAutoSessionTracking` flag for the users, and the capacitor init just determines to enable native or browser session tracking without needing the user to know about it.